src/content/docs/auto-telemetry-pixie/get-started-auto-telemetry-pixie.mdx

```diff
@@ -30,6 +30,8 @@ You’ll get visibility into HTTP services using golden signals, HTTP transactio
 
 And then dive deeper using embedded visualizations of your Pixie data. Quickly identify hot spots with [Flamegraph](#flamegraph). On the [Live debugging with Pixie tab](#live-debugging), answer questions like what SQL requests your app is making or which services are talking to each other. 
 
+Click **Open in IDE** to use CodeStream to find the source of the usage spike in your IDE.
+
```

Comment:
Dave (@planteater) We probably need to say more here and link to a how-to on setup.  I havn't been tracking what we plan to add for this but let me know if you need help tracking that down.

Reply:
If it's not going to make it in for launch, I think I should just remove mention of it altogether both here and in the other docs where I've mentioned this. @planteater